generated/ghes-2.18.json

```diff
@@ -39429,6 +39429,7 @@
           },
           "body_html": { "type": "string" },
           "body_text": { "type": "string" },
+          "mentions_count": { "type": "integer" },
```

Comment:
confirming that `release.mentions_count` does in fact exist in all GHES versions